package.json

```diff
@@ -21,7 +21,7 @@
 	"browser": "./dist/browser/extension",
 	"main": "./dist/src/extension",
 	"engines": {
-		"vscode": "^1.75.0",
```

Comment:
Unless I'm missing something, I don't think we need to bump vscode engine in this pull request?

Reply:
Updated based on the following error when packaging:
 ERROR  @types/vscode ~1.89.0 greater than engines.vscode ^1.75.0. Either upgrade engines.vscode or use an older @types/vscode version.

Changing the version of the types to avoid regressions.